### PR TITLE
fix(tier4_state_rviz_plugin): split into two panels

### PR DESCRIPTION
**Pull request by h-ohta**
## Description

I split autoware state panel into two. in order to resolve inconvenient for rviz.
image

## Related Links

TIER IV INTERNAL LINK: https://tier4.atlassian.net/browse/T4PB-26296

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.cpp [hunk 1]
```diff
@@ -0,0 +1,324 @@
+//
+//  Copyright 2020 Tier IV, Inc. All rights reserved.
```
Comment (isamu-takagi) [inline common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.cpp:2]:
```suggestion
//  Copyright 2023 Tier IV, Inc. All rights reserved.
```

File: common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.hpp [hunk 2]
```diff
@@ -0,0 +1,68 @@
+//
+//  Copyright 2020 Tier IV, Inc. All rights reserved.
```
Comment (isamu-takagi) [inline common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.hpp:2]:
```suggestion
//  Copyright 2023 Tier IV, Inc. All rights reserved.
```

File: common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.cpp (hunk 1 above)
Comment (takayuki5168) [reply common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.cpp]: Also "TIER IV"

File: common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.cpp [hunk 3]
```diff
@@ -0,0 +1,324 @@
+//
+//  Copyright 2023 Tier IV, Inc. All rights reserved.
```
Comment (h-ohta) [inline common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.cpp:2]:
```suggestion
//  Copyright 2023 TIER IV, Inc. All rights reserved.
```

File: common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.hpp [hunk 4]
```diff
@@ -0,0 +1,68 @@
+//
+//  Copyright 2023 Tier IV, Inc. All rights reserved.
```
Comment (h-ohta) [inline common/tier4_state_rviz_plugin/src/velocity_steering_factors_panel.hpp:2]:
```suggestion
//  Copyright 2023 TIER IV, Inc. All rights reserved.
```

**Review by takayuki5168 — APPROVED** [review]

**Review by isamu-takagi — APPROVED** [review]

**Pull request merged**